blog - lessons from Elon Musk (#7571)
PostHog/posthog.com: contents/blog/learnings-from-elon.md

---
date: 2024-01-20
title: Learnings from Elon Musk
rootPage: /blog
sidebar: Blog
showTitle: true
author:
  - james-hawkins
hideAnchor: true
category: Startups
---

People hate or love him. Regardless, I read the Elon Musk biography by Walter Isaacson over Christmas and here are some lessons that we can take from his work.

## The power (or not) of artificial deadlines and urgency

> There are many examples of Elon setting a ridiculous seeming artificial deadline to create urgency. This clearly works for him - look at all the progress he has made.

Author: add examples

Second, we think more experienced people are usually better at working proactively - since they get stuck less often and need less help. These people generally do not want to work until midnight on a weekend, as they usually, but not always, have families because they’re normally older. We’ve gone so far as to have an all remote environment, which is particularly well suited to people like this (amongst many other benefits, such as long, uninterrupted blocks of time to just build cool stuff).

Third, we grow by shipping, not by hiring salespeople. This means we need to optimize for long term shipping speed. If we set artificial feature-focused deadlines, we’ll incur technical debt, which will speed us up in the short run, but will hurt our long term growth rate.

To broaden this lesson, I think you need to either be all in on working 28 hours a day, 9 days a week with a bunch of 20 year olds, or you should focus on a smaller, more experienced team. The former suits artificial deadlines, the latter sees them as a bad thing.

## Product drives marketing and sales

> There is a very product-first approach - many of his companies put something so cool on the internet, that it generates an extreme amount of word of mouth growth.

Author: link to some cool tweets of the cybertruck or whatever else

We grow primarily through word of mouth. Long term, we think this is a winning strategy as it’s more efficient than a sales led model - you can reinvest everything back into the product, to build a bigger and bigger long term advantage.

You can see this from the mind blowing specification of the Teslas, for example - they are incredibly powerful, superb value and very cool. There’s really no major compromise. Hence they rocketed in popularity.

> In the details but not the day to day running

Author: ```suggestion
## In the details but not the day to day running
```

We do not think this is a strength - perhaps it works in his companies because it creates a fear element, and more urgency as a result. 

At PostHog, as mentioned, we’re more intrinsically motivated so ruling by fear would be harmful to performance.

Whilst Tim and I technically are the CTO and CEO respectively, we more or less share the CEO role. We’ve been very effective like this. I think it’s because we’re a product led company, so what we ship has a very strong impact on marketing and sales (see “Product drives marketing and sales” above), so keeping a tight connection between these varied teams really matters.

The key to making a split like this work is happily spending a lot of time together. This (and being ok sometimes missing out on decisions) is the only way to be decisive when two of you are paired like this. We talk a lot (several hours a week, minimum) - which can feel inefficient, but the reality is we need to optimize for organizational effectiveness, not our own schedules.

## Likes the taste of his own blood

> Clickbait subtitle. You got me. Elon gives very direct feedback and makes painful decisions rapidly and doesn’t dwell on any of this stuff. For example, he 

Author: finish the sentence

Reviewer: bought twitter? https://www.nbcnews.com/business/business-news/twitter-elon-musk-timeline-what-happened-so-far-rcna57532

We describe PostHog as a team for this reason. We have performance managed - letting people go decisively - throughout our time and it has been one of the most important things we’ve ever done.

We do this because we’re optimizing for our strongest performers, who want to work in an environment where everyone is shipping their best work. As soon as we make it ok not to push hard, everyone will feel it and eventually quit or reduce their own input.

We try to balance this by being as kind as possible for people who depart under these circumstances - we pay four months’ severance once someone has been here more than a couple of months. Beyond pay, we talk directly and honesty to people - we avoid doing PIP (Performance Improvement Program) type processes as we think they are dehumanizing on both sides and cause more legal cases than they solve.

## Fun is a serious business

> There’s very little written in startup lore about the importance of having fun. Elon seems to get a clear kick of out what he does.

Author: the way he uses twitter / buys it?!

> We often make decisions at PostHog based on what will be the most entertaining. This can lead to remarkable things happening with our website, marketing, customer success and product. The reason we do this is that we want to build all the products in one. That will take a long time as it’s a lot of products, so we need to consider PostHog a marathon, and fun keeps people’s spirits up.

Author: explain a bit more about how this was a factor in self host -> cloud focus

> We often make decisions at PostHog based on what will be the most entertaining. This can lead to remarkable things happening with our website, marketing, customer success and product. The reason we do this is that we want to build all the products in one. That will take a long time as it’s a lot of products, so we need to consider PostHog a marathon, and fun keeps people’s spirits up.

Reviewer: ```suggestion
We often make decisions at PostHog based on what will be the most entertaining. This can lead to remarkable things happening with our website, marketing, customer success and product. The reason we do this is that we have a lot of products to build which will take a long time. We need to consider PostHog a marathon, and fun keeps people’s spirits up.
```

## It’s easier when it’s harder

> Elon’s companies are wildly ambitious - to make humans a multi-planetary species, for example.

Author: maybe list every company and its mission to make this piece less space-x centric

Reviewer: I think you touch on the others ones so this is fine.

In the early days of PostHog, we were a self hostable version of product analytics. That was very pragmatic - it filled a clear market need, but it wasn’t world changing.

As we got bigger, we created a more inspiring mission, alongside a slightly outrageous strategy.

We help engineers build better products, and the first way we do this is by giving them all the products they need to do this.

Yes, our strategy is to build a ton of potential products. We have built 8 already, and have a list of 30 more. It may end up being hundreds. 

As we got more direct with this strategy - putting it everywhere on the website, in our job ads, and in our all hands, it became self fulfilling. Engineers spotted how challenging it looks to do this, and this draws the very best people in. We now get hundreds or even thousands of applicants for every job role, and the product itself is so impressive that our main way of growing is word of mouth. 

## His businesses are real companies

> Take the example of SpaceX - despite a very futuristic vision, that extends so far, Elon may not live to really see it, they’re heads are firmly below the clouds. They created a mission and then figured out a business model - shipping things into space for money, which could fuel the mission.

Author: ```suggestion
Take the example of SpaceX - despite a very futuristic vision, that extends so far, Elon may not live to really see it, their heads are firmly out of the clouds. They created a mission and then figured out a business model - shipping things into space for money, which could fuel the mission.
```

> At PostHog, are disciplined with each product we ship - we care about performance, margins, and revenue  by product. Many venture capitalists will tell you to hire like crazy when you have product market fit. We have product market fit, but we don’t do this at all. We take pride in our individual products being real businesses - potentially profitable, with revenue and with sane margins.

Reviewer: ```suggestion
At PostHog, are disciplined with each product we ship - we care about performance, margins, and revenue. Many venture capitalists will tell you to hire like crazy when you have [product market fit](/founders/product-market-fit-game). We have product market fit, but we don’t do this at all. We take pride in our individual products being real businesses - potentially profitable, with revenue and with sane margins.
```
